x/id/internal/types/codec.go

```diff
@@ -6,13 +6,6 @@ import (
 	"github.com/cosmos/cosmos-sdk/codec"
 )
 
-/**
- * Any interface you create and any struct that implements an interface needs to be
- * declared in the RegisterCodec function.
- * In this module the Msg implementations (SetIdentity) need to be registered,
- * but your Identity query return type does not.
- */
-
 // RegisterCodec registers concrete types on wire codec
 func RegisterCodec(cdc *codec.Codec) {
 	cdc.RegisterConcrete(MsgSetIdentity{}, fmt.Sprintf("%s/SetIdentity", ModuleName), nil)
```

Comment:
As you told me to do, removing `fmt.Sprintf` shouldn't be a problem at all, leaving only **commercio/SetIdentity** instead.

Reply:
Should've been fixed now